tox.ini

```diff
@@ -60,10 +60,12 @@ basepython = python3.8
 skip_install= True
 deps =
     flake8
+    mypy
     isort
     black
 commands = 
     flake8 jose setup.py
+    mypy jose setup.py
```

Comment:
Should `mypy` not be added to the dev requirements?

Reply:
Do you mean `requirements-dev.txt`?
It was not added following `flake8`, `isort` and `black` example

Reply:
Yes that is what I meant.

> It was not added following `flake8`, `isort` and `black` example

I did not notice that, probably have to leave it up to the maintainers then :)